build.gradle

```diff
@@ -33,6 +35,24 @@ allprojects {
   repositories {
     mavenCentral()
   }
+
+  apply plugin: 'com.github.ben-manes.versions'
+
+  dependencyUpdates {
+    revision="release"
+    resolutionStrategy = {
+      componentSelection { rules ->
+        rules.all { ComponentSelection selection ->
+          boolean rejected = ['snap', 'alpha', 'beta', 'rc', 'cr', 'm'].any { qualifier ->
```

Comment:
This section is rejecting any dependencies that contain these strings to prevent "non-release" artifacts from getting into the report. The 'm' is to filter out release candidates for Scala. I think it stands for "Milestone":
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.scala-lang%22%20AND%20a%3A%22scala-library%22